src/test/scripts/check_enterprise_merge.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# fail if trying to reference a variable that is not set.
+set -u
+# exit immediately if a command fails
+set -e
+
+# try_merge sees if we can merge "from" branch to "to" branch
+# it will exit with nonzero code if the merge fails because of conflicts.
+try_merge() {
+    to=$1
+    from=$2
+    git checkout "${to}"
+    # this will exit since -e option is set and it will return non-zero code on conflicts.
+    git merge --no-ff --no-commit "${from}"
+
+}
+
+git config --global user.email "citus-bot@microsoft.com" 
+git config --global user.name "citus bot" 
+
+cd ~
+git clone https://${GIT_USERNAME}:${GIT_TOKEN}@github.com/citusdata/citus-enterprise
+cd citus-enterprise
+git fetch --all
+
+# echo commands
+set -x
+git branch -r --list
+
+branch_name="${CIRCLE_BRANCH}"
+
+if [ `git branch -r --list origin/$branch_name` ]
```

Comment:
Does this check the error code or the output? Could maybe be clarified.

Reply:
it checks the output, if the branch doesn't exist the output will be empty.